### PR TITLE
DOC: Correct usage example for np.char.decode docstring

### PR DESCRIPTION
**Pull request by BrentTanKian**

### CODE REVIEW

File: numpy/core/defchararray.py [hunk 1]
```diff
@@ -528,8 +528,8 @@ def _code_dispatcher(a, encoding=None, errors=None):
 
 @array_function_dispatch(_code_dispatcher)
 def decode(a, encoding=None, errors=None):
-    """
-    Calls `str.decode` element-wise.
+    r"""
+    Calls `bytes.decode` element-wise.
```
Comment (seberg) [inline numpy/core/defchararray.py:532]:
```suggestion
    Calls ``bytes.decode`` element-wise.
```

File: numpy/core/defchararray.py [hunk 2]
```diff
@@ -2205,7 +2205,7 @@ def count(self, sub, start=0, end=None):
 
     def decode(self, encoding=None, errors=None):
         """
-        Calls `str.decode` element-wise.
+        Calls `bytes.decode` element-wise.
```
Comment (seberg) [inline numpy/core/defchararray.py:2208]:
```suggestion
        Calls ``bytes.decode`` element-wise.
```

**Review by seberg — APPROVED** [review]
LGTM, I will apply the small change that Matti had added before to use ``` `` ``` (double backticks).  That ensures that this is rendered as code rather than a (broken) link.  The alternative would be to add that `:py:meth:` part to make it work.

(We have a lot of broken links in the docs, so that is not a big deal even if it was not changed.)

Otherwise, will just wait for the tests, thanks @BrentTanKian

**Comment by BrentTanKian** [comment]
Thanks so much @seberg :)

**Pull request merged**